common/stats/disk_monitor.go

```diff
@@ -0,0 +1,97 @@
+package stats
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// DiskMonitor monitor disk usage for selected directories
+type DiskMonitor struct {
+	shortName  []string
```

Comment:
what's a shortName

Reply:
short name is the name the suffix that is appended to the disk_usage metric name.  from the test: short name is **cwd**, the metric is reported as **directorySizeChange_kb_cwd**.  I think I'll change it to: **commandDirUsageKb_cwd**. wdyt

Reply:
sgtm!